modules/imgcodecs/src/utils.hpp

```diff
@@ -138,6 +138,8 @@ CV_INLINE bool  isBigEndian( void )
     return (((const int*)"\0\x1\x2\x3\x4\x5\x6\x7")[0] & 255) != 0;
 }
 
+FILE *fopen(const char *fn, const char *mode);
```

Comment:
this is not a good practice to manually declare standard functions. Use `#include <stdio.h>` instead